openslides_backend/action/actions/meeting/import_.py

```diff
@@ -217,6 +217,15 @@ def update_instance(self, instance: dict[str, Any]) -> dict[str, Any]:
                     "last_login",
                     "committee_ids",
                 ],
+                "motion_workflow": [
+                    "default_statute_amendment_workflow_meeting_id",  # This is removed due to deprecation of statutes
+                ],
```

Comment:
see PR #2485: In fact I would prefer a migration if there is or will be one for removed field

Reply:
Migration is now available.